# ArrowDetection3D.msg
Header header
string label
float32 confidence
float32 x
float32 y
float32 z
